pkg/application/registry/application/storage/storage.go

```diff
@@ -66,6 +66,8 @@ func NewStorage(optsGetter genericregistry.RESTOptionsGetter,
 		CreateStrategy: strategy,
 		UpdateStrategy: strategy,
 		DeleteStrategy: strategy,
+
+		ShouldDeleteDuringUpdate: applicationtrategy.ShouldDeleteDuringUpdate,
```

Comment:
applicationtrategy --> applicationstrategy ?

Reply:
miss a "s"

Reply:
no，applicationtrategy is right